### ISSUE TITLE
Don't do automatic line breaks within LaTeX

### ISSUE THREAD

**Issue by cobordism** [ManimCommunity/manim#475]
### Context
If we make a long line of text `Tex("this is a long line of text... perhaps")` then the output will contain a line break if the line in question is longer than a full line in the tex document (as defined by the tex template). It bears absolutely no relation to the width of the screen in manim.
We have plenty of options for breaking up strings in manim and arranging and aligning  them left to right or top to bottom ... we can either use LaTeX for layout or we can use manim. It is unclean if we use both.

Now if you encounter a line-break in LaTeX that you do not want, you have work to do. 
You can try to change your code to `Tex("line1", "line2")` and then arrange the two parts by hand (but the spacing will be off).
You can change your tex code (for example including everything in an \mbox) but you'd no longer be able to break up the mobject by substrings.

### Suggested improvement 
I suggest that we include in the default text template the geometry package, specifying an extremely wide page width so that we avoid LaTeX based automatic line breaks entirely - manual line breaks will of course work without issue. (More info perhaps here https://tex.stackexchange.com/questions/35753/how-create-a-paper-of-x-y-dimensions-size ).

One might be tempted to say we should just try to match the page width in TeX page width the video width in manim, but this is a fool's errand as we can and do scale mobjects within manim after they have left the tex->dvi->svg->mobject pipeline.

Alternate idea: make it a keyword argument to Tex() and let users play around with the value depending on the needs of their scene.

**Comment by leotrs** [comment]
My first instinct is always to provide more customization options, if possible, with a clean interface. So I'd vote for setting the page geometry to fit the manim canvas by default, but allow the user to change it (say to an extremely wide page) easily if they want to.

**Comment by cobordism** [comment]
which raises the question... how many centimetres wide is the default maim screen? 
and what role does `TEX_MOB_SCALE_FACTOR` play?

**Comment by leotrs** [comment]
Uuuuh, that's an excellent question. We have a default measurement in pixels, not centimeters.

I have no idea what TEX_MOB_SCALE_FACTOR is.

**Comment by MysaaJava** [comment]
May be linked to #275 as centimeters creates a coordinates system too

**Comment by leotrs** [comment]
Yes, I should have said that I would friggin love to have this, but it doesn't currently exist rn.

**Comment by vaclavblazej** [comment]
In some capacity may be linked to #1921 as long lines would solve that issue and textfield's width seems to have no correspondence to the video width either.
